draft-ietf-quic-transport.md

```diff
@@ -92,7 +92,10 @@ informative:
 
 --- abstract
 
-This document defines the core of the QUIC transport protocol.  Accompanying
+This document defines the core of the QUIC transport protocol. QUIC provides
+multiplexed streams for delivering ordered application data, flow control,
+low-latency connection establishment, migration, resilience to path changes,
```

Comment:
For the abstract, I feel like it's sufficient to let migration and path changes be the same thing.

Reply:
Agreed.  I've tweaked this.  And the next sentence too, mainly to avoid a possessive.